src/client/src/ui/styles/scss/index.scss

```diff
@@ -20,4 +20,5 @@
 @import 'popoutRegion.scss';
 @import 'spotTileControls.scss';
 @import 'variables';
+@import 'Alert-notification.scss';
```

Comment:
alertNotification.scss for consistency

Reply:
For some strange reason, this broke the import! Will take a look again

Reply:
Done